src/main/scala/top/Configs.scala

```diff
@@ -79,7 +79,7 @@ class MinimalConfig(n: Int = 1) extends Config(
         StoreQueueNWriteBanks = 4, // NOTE: make sure that StoreQueueSize is divided by StoreQueueNWriteBanks
         StoreQueueForwardWithMask = true,
         // ============ VLSU ============
-        VlMergeBufferSize = 8,
+        VlMergeBufferSize = 16,
```

Comment:
Why is `VlMergeBufferSize` modified?

Reply:
vlsuCanAccept needs both vsSplit(i) and VlSplit(i) to be ready. Under MinimalConfig, if there is a vload instruction before executing the vstore instruction, which splits many uops, vlMergeBuff is full, causing the vstore instruction to fail to execute.